### PR TITLE
✨Add "conative" ad network support to amp-ad

### PR DESCRIPTION
**Pull request by mkdefinemedia**
This adds "conative" ad network support to amp-ad.

### CODE REVIEW

**Comment by mkdefinemedia** [comment]
fyi: i'm still waiting for the CLA

File: ads/_config.js [hunk 1]
```diff
@@ -351,6 +351,10 @@ const adConfig = jsonConfiguration({
     prefetch: 'https://static.clmbtech.com/ad/commons/js/colombia-amp.js',
   },
 
+  'conative': {
+    renderStartImplemented: true,
```
Comment (calebcordry) [inline ads/_config.js]: verified `renderStart` is being called. 👍

File: ads/conative.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function conative(global, data) {
+  validateData(data, ['domain', 'adslot', 'height']);
```
Comment (calebcordry) [inline ads/conative.js]: You are able to add optional params as a third arg to this function. Would you like to include `preview` here?

File: ads/conative.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Example
+
+- All CONATIVE `<amp-ad>` tags require the `width`, `height`, `layout`, `sizes` and `type="conative"` parameters.
+
+```html
+  <amp-ad 
+    height="250"
+    width="300"
+      type="conative"
```
Comment (calebcordry) [inline ads/conative.md]: nit: can you indent all these to the same level?

Comment (mkdefinemedia) [reply ads/conative.md]: sorry, my bad! done!

**Comment by calebcordry** [comment]
Thanks for the contribution!

**Pull request merged**